### ISSUE TITLE
Train Error report in CUHK-SYSU

### ISSUE THREAD

**Issue by aaagx** [LukeJaffe/GFN#2]
Hi  I have tried train GFN in my own device(Tesla T4  16G) in cuhk dataset by using 
osr_run --trial_config=./configs/cuhk_train_final.yaml

just set 
batch size : 4  
emb_norm_type: 'layernorm' (use batchnorm may also occur propblem like  [error.txt](https://github.com/LukeJaffe/GFN/files/9967785/error.txt))

and i got a problem in epoch 22
[error.txt](https://github.com/LukeJaffe/GFN/files/9967770/error.txt)

**Comment by aaagx** [comment]
In PRW dataset，all the things is ok

**Comment by dustasa** [comment]
File "/home/ubuntu/conda/envs/osr/lib/python3.7/site-packages/torch/nn/functional.py", line 2404, in _verify_batch_size
    raise ValueError("Expected more than 1 value per channel when training, got input size 
Go to `functional.py` and commit out the following two lines, it'll work.

**Comment by dustasa** [comment]
File "/home/ubuntu/conda/envs/osr/lib/python3.7/site-packages/torch/nn/functional.py", line 2404, in _verify_batch_size
raise ValueError("Expected more than 1 value per channel when training, got input size
Go to functional.py and comment out the following two lines, it'll work.

**Comment by LukeJaffe** [comment]
Thanks for pointing this out. It looks like this is caused by a rare case where the batch size is 1 for the embeddings, unlucky combination of augmentations removes most boxes from the input scenes. I will add in a manual check for this case, and simply use the running batch statistics when the batch size is 1.

Also, I recommend using BatchNorm instead of LayerNorm for better performance.

**Comment by aaagx** [comment]
I  have tried  train again ,It has been 27  epochs now, everything is ok
